go/buildhelper/buildconstants/constants.go

```diff
@@ -0,0 +1,140 @@
+package buildconstants
+
+import "github.com/ethereum/go-ethereum/common"
+
+const ADDR1PK = "5dbbff1b5ff19f1ad6ea656433be35f6846e890b3f3ec6ef2b2e2137a8cab4ae"
+const ADDR2PK = "b728cd9a9f54cede03a82fc189eab4830a612703d48b7ef43ceed2cbad1a06c7"
+const ADDR3PK = "1e1e76d5c0ea1382b6acf76e873977fd223c7fa2a6dc57db2b94e93eb303ba85"
+
+var (
+	CONTRACT_ADDRESS = common.HexToAddress("0x63eA4F258791234606F9d11C0C66D83c99bfC7CA")
```

Comment:
@otherview Do we need to add a TODO to say that this needs to be dynamic eventually? You'd want to configure this rather than have it hardcoded, right?

Reply:
Good point. Eventually, it will be an input of sorts.